From 681b78750459e1ba9e1211501848676e954bcc86 Mon Sep 17 00:00:00 2001
From: Dan Cartier <nospamdan1@gmail.com>
Date: Thu, 24 Oct 2019 10:14:49 -0400
Subject: [PATCH] build: Don't check for fingerprint mismatch

* This removes the warning that sidplays at boot which states:
  "There is an internal problem with your device.  Contact the manufacturer for details"

Signed-off-by: Dan Cartier <nospamdan1@gmail.com>
Change-Id: I5515f204bcdbe0a792f95aa1ab321fe4a8fd2696
Signed-off-by: Joey Huab <joey@evolution-x.org>
---
 core/java/android/os/Build.java | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/core/java/android/os/Build.java b/core/java/android/os/Build.java
index b7dd36e2c4f5..1468bff9dcc0 100755
--- a/core/java/android/os/Build.java
+++ b/core/java/android/os/Build.java
@@ -1221,7 +1221,7 @@ public class Build {
      */
     public static boolean isBuildConsistent() {
         // Don't care on eng builds.  Incremental build may trigger false negative.
-        if (IS_ENG) return true;
+        /*if (IS_ENG) return true;
 
         if (IS_TREBLE_ENABLED) {
             // If we can run this code, the device should already pass AVB.
@@ -1234,6 +1234,7 @@ public class Build {
             }
 
             return result == 0;
+            return true;
         }
 
         final String system = SystemProperties.get("ro.system.build.fingerprint");
@@ -1256,7 +1257,7 @@ public class Build {
                         + " but vendor reported " + vendor);
                 return false;
             }
-        }
+        } */
 
         /* TODO: Figure out issue with checks failing
         if (!TextUtils.isEmpty(bootimage)) {
-- 
2.32.0

